### DISCUSSION TITLE
有多个监听函数冲突时该怎么办？

### DISCUSSION THREAD

**Discussion by worklia** [simple-robot/Discussions#133] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

比如我有一个全局的监听，好友输入任何内容没有匹配的就会被这个函数监听，有匹配的监听函数就不会调用这个全局的监听。请问能否实现，我在@Listener注解中看到了能设置优先级，但是我不知道如何使用

**Comment by ForteScarlet** [comment, marked as answer]
https://github.com/orgs/simple-robot/discussions/62
